.github/continuous-integration.yaml

```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
+
+on: pull_request
+
+env:
+  DOCKER_BUILDKIT: 1
+
+jobs:
+  docker:
+    name: Build docker images
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: build php
+        uses: docker/build-push-action@v1
+        with:
+          path: ./php
```

Comment:
Looks like there's an issue on the action https://github.com/docker/build-push-action/issues/29